themes/gauge_theme/static/js/theme.js

```diff
@@ -89,5 +89,44 @@ $(document).ready(() => {
         scrollTop : 0                       // Scroll to top of body
     }, 500);
   });
+
```

Comment:
Do we have to repeat this js here?
Can't we pull it from `https://preview.gauge.org/themes/gauge_theme/static/js/theme.js` ?

Reply:
Hi Zabil, reason I am using Js here is It will effect load time If pull it from preview.gauge

Reply:
Don't think so, both are hosted on the AWS buckets in the same region. It will be cached.